lib/pf/web/constants.pm

```diff
@@ -93,18 +93,15 @@ Readonly::Scalar our $URL_GAMING_REGISTRATION   => '/gaming-registration';
 Readonly::Scalar our $URL_DEVICE_REGISTRATION   => '/device-registration';
 Readonly::Scalar our $URL_DEVICE_REG_LOGOUT     => '/device-registration/logout';
 
-# External Captive Portal detection constant
-Readonly::Scalar our $REQ_CISCO                 => 'ap_mac';
-Readonly::Scalar our $REQ_MAC                   => 'mac';
-Readonly::Scalar our $REQ_ARUBA                 => 'apname';
-Readonly::Scalar our $REQ_CISCO_PORTAL          => '/cep(.*)';
-Readonly::Scalar our $REQ_RUCKUS                => 'sip';
-Readonly::Scalar our $REQ_AEROHIVE              => 'RADIUS-NAS-IP';
-
 # External Captive Portal URL detection constant
-Readonly::Scalar our $EXT_URL_XIRRUS            => '^/Xirrus::AP_http';
-Readonly::Scalar our $EXT_URL_MERAKI            => '^/Meraki::AP_http';
-Readonly::Scalar our $EXT_URL_COOVACHILLI       => '^/CoovaChilli';
+Readonly::Scalar our $EXT_URL_AEROHIVE              => '^/AeroHIVE::AP';
+Readonly::Scalar our $EXT_URL_ARUBA                 => '^/Aruba';
+Readonly::Scalar our $EXT_URL_CISCO_CATALYST_2960   => '^/Cisco::Catalyst_2960';
```

Comment:
Arf ok, this is because you instantiate the switch module with that ... so forget and let's use a long name.